openmetrics/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "categories": [
+    "monitoring"
+  ],
+  "creates_events": false,
+  "display_name": "OpenMetrics",
+  "guid": "3f67af75-6987-468c-99b3-5001ba5ab414",
+  "is_public": true,
```

Comment:
If this integration is shipped with agent 6.6, then this parameter needs to be set to `false` in order to avoid confusion.

Reply:
and we need to set it back to true when agent 6.6 is GA